### PR TITLE
feat: show a video to be played that is loaded from YouTube in an embed

### LINKED ISSUE #160: feat: show a video to be played that is loaded from YouTube in an embed
show a video to be played that is loaded from YouTube in an embed

This is part of ** feat: create play video component route** #143

### PR DESCRIPTION
**Pull request by psyhedeliq**
this fixes #160

### CODE REVIEW

**Review by QuintyHH — APPROVED** [review]
:100:

File: src/component/lecture-info/lecture-info.component.jsx [hunk 1]
```diff
@@ -6,8 +6,10 @@ import { StyledLectureInfo } from './lecture-info.style';
 const LectureInfo = ({ title, id, description, imagePath, youtubeVideoId, youtubePlaylistId }) => {
   const getImagePath = () =>
     imagePath ? imagePath : `http://img.youtube.com/vi/${youtubeVideoId}/0.jpg`;
-  const getToutubePath = () =>
-    `https://www.youtube.com/watch?v=${youtubeVideoId}&list=${youtubePlaylistId}&t=0s`;
+  const getToutubePath = () => `/video/${youtubeVideoId}`;
```
Comment (marianzburlea) [inline src/component/lecture-info/lecture-info.component.jsx]: `getToutubePath`?

File: src/component/play-video/play-video.component.jsx [hunk 2]
```diff
@@ -1,7 +1,16 @@
 import React from 'react';
 
 const PlayVideo = ({ youtubeVideoId }) => {
-  return <h1>VIDEO: {youtubeVideoId}</h1>;
+  return (
+    <iframe
+      width="560"
```
Comment (marianzburlea) [inline src/component/play-video/play-video.component.jsx]: is this a good approach to do fixed width?
I suggest to use 100% width and a ratio of 16/9 to make it fluid and dynamic

**Review by marianzburlea — CHANGES_REQUESTED** [review]

**Review by marianzburlea — APPROVED** [review]
:100:

**Review by tandaraelena — APPROVED** [review]
looks good 👍

**Review by QuintyHH — APPROVED** [review]
Byebye Toutube

**Review by valentin3199 — APPROVED** [review]

**Pull request merged**